doc/userguide/install.rst

```diff
@@ -170,18 +170,85 @@ Follow these steps from your Suricata directory::
 Binary packages
 ---------------
 
-Ubuntu
-^^^^^^
+Ubuntu from Personal Package Archives (PPA)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Wasn't sure where this should go, but this seemed like the best place, from what already exists.